### ISSUE TITLE
$.my.tojson exhaust stack in rear cases

### ISSUE THREAD

**Issue by ermouth** [ermouth/jQuery.my#25]
Regexps in this fn https://github.com/ermouth/jQuery.my/blob/master/Release/1.2/jquerymy-1.2.4.js#L727 may produce enormously deep recursion and exhaust js machine stack.

Minor bug, this function is never used during normal runtime.

**Comment by ermouth** [comment]
Takes place in Chrome cause V8 reflector behaves in a strange way, and this behavior seems to vary from version to version:

```
(new Function('a','return a')).toString() -->
"function anonymous(a
/**/) {
return a
}"
```
but
```
(function(a){return a}).toString() -->
"function (a){return a}"
```

**Comment by ermouth** [comment]
Fixed in 1.2.5
